docs/blog/introduction-0.4.0/index.qmd

```diff
@@ -0,0 +1,137 @@
+---
+title: "Great Tables `v0.4.0`: Nanoplots and More"
+html-table-processing: none
+author: Rich Iannone
+date: 2024-03-19
+freeze: true
+jupyter: python3
+---
+
+The recent `v0.4.0` release of **Great Tables** contains nanoplots as a major new feature. So, in this post I'll concentrate on showing you all the things you can do with nanoplots. While the emphasis here will be on the aforementioned nanoplots feature, the last two releases (`v0.3.1` and `v0.4.0`) overall gave us a nice collection of improvements which includes:
+
+- the [`fmt_nanoplot()`](`great_tables.GT.fmt_nanoplot`) method for adding nanoplots to your table
+- improved HTML table representations in different code environments
+- integration of Polars selectors in the `columns=` arg of all formatting (`fmt_*()`) methods
+- the [`save()`](`great_tables.GT.save`) method for saving a **GT** table as an image file
+- rendering a **GT** table as an HTML string though [`as_raw_html()`](`great_tables.GT.as_raw_html`)
+
+Now let's dive into the wonderful world of nanoplots!
```

Comment:
It would be nice to show a nanoplot here without code (or expandable code). That way, people can see the result right away, before putting effort into reading code.

Is there a good nanoplot example we can use to make a splash (e.g. something that might also live on the examples page)?